bpf/lib/signal.h

```diff
@@ -50,4 +51,15 @@ static __always_inline void send_signal_nat_fill_up(struct __ctx_buff *ctx,
 	send_signal(ctx, &msg);
 }
 
+static __always_inline void send_signal_ct_fill_up(struct __ctx_buff *ctx,
+						   __u32 proto)
+{
```

Comment:
Too late, but maybe it makes sense to send a signal only `ifndef HAVE_LRU_HASH_MAP_TYPE`?

Reply:
It is very unlikely, but for LRU there is a corner case where it could also return an ENOMEM error on prealloc when there is nothing in the pending list.

Reply:
Discussed offline: it shouldn't increase the verifier complexity too much, so it's fine to unconditionally enable the sending.